draft-ietf-rats-endorsements.md

```diff
@@ -201,13 +201,33 @@ some minimal matching policy (e.g., exact match against a singleton reference
 value).  This unfortunately complicates design as a Verifier may need
 multiple parsers for matching policies.
 
-# Endorsing Identity
+# Key Material for Attestation
 
-One type of claims that might be endorsed would be claims having to do with
-identity, such as verification keys bound to an identifier for the Attesting Environment (e.g., {{-rats-eat}} `ueid`).  While identity claims are just another
+A Verifier receives Evidence from an Attester that must be considered untrusted
+until established otherwise. This trust is established using cryptography. Typically,
+the bottom-most Attesting Environment in an Attester will sign claims about a Target Environment
+with a private key that that Attesting Environment possesses and the Verifier will verify
+them with a public key it possesses, called a verification key below. While this is typical,
+cryptography other than public key may also be used.
```

Comment:
This sentence need healing.

Reply:
Can you elaborate on what the problem is?

Reply:
I read the text as a whole a few times and it seems just to be unusual English phrasing for me. The new text is pretty terse, but it seems to be good enough.

In general in this Section Attester and Attesting Environment seem to be used synonymously. Would it break the meaning of the text if we would change "Attester" to "Attesting Environments of an Attester"? See exemplary suggestion above https://github.com/ietf-rats-wg/rats-endorsements/pull/18/files#r1625613299